test/integration/scenarios/network/peer_disconnect.js

```diff
@@ -133,8 +122,8 @@ module.exports = params => {
 				});
 
 				it('start all nodes that were stopped', done => {
-					for (let i = 1; i < totalPeers; i++) {
-						startNode(`node_${i}`);
+					for (let i = 1; i < networkParameters.TOTAL_PEERS; i++) {
+						common.startNode(`node_${i}`);
 					}
 					setTimeout(() => {
```

Comment:
Same as above

Reply:
Stop node calls syncStop node calls synchronous `childProcess.execSync`, so it's synchronous as well. Later on we are waiting 2000 ms to ensure the process finished though. The proper solution would have to change childProcess.execSync to childProcess.exec and utilize the output. 
It can be added as a separate issue as the next following improvements.hronous `childProcess.execSync`, so it's synchronous as well. Later on we are waiting 2000 ms to ensure the process finished though. The proper solution would have to change childProcess.execSync to childProcess.exec and utilize the output. 
It can be added as a separate issue as the next following improvements.